redaxo/src/addons/structure/plugins/content/tests/article_slice_test.php

```diff
@@ -7,7 +7,7 @@
  */
 class rex_article_slice_test extends TestCase
 {
-    private const FAKE_ID = 4294967295;
+    private const FAKE_ID = 2147483647; // max int on 32bit
```

Comment:
hast du bewusst nicht `PHP_INT_MAX` genommen?

Reply:
das wäre auf 64bit systemen eine zu große zahl für `int(10) unsigned` in mysql.